lib/functions-filters.php

```diff
@@ -45,3 +45,12 @@ function add_lazysize_on_srcset($attr) {
 
 }
 add_filter('wp_get_attachment_image_attributes', 'add_lazysize_on_srcset');
+
+// Allow for OGV file type uploads
+add_filter( 'upload_mimes', 'allowed_filetypes', 1, 1 );
```

Comment:
> $priority
(int) (Optional) Used to specify the order in which the functions associated with a particular action are executed. Lower numbers correspond with earlier execution, and functions with the same priority are executed in the order in which they were added to the action.
Default value: 10
$accepted_args
(int) (Optional) The number of arguments the function accepts.
Default value: 1